docs/content/readme.md

```diff
@@ -47,7 +47,8 @@ See the [Getting Started setup guide](direct/get-started.md) for getting started
 
 ## Contributing
 
-We ❤️ our contributors! If you're interested in helping us out, please head over to our [Contributing](Contribution%20guidelines/CONTRIBUTING.md) guide.
+
+We ❤️ our contributors! If you're interested in helping us out, please head over to our [Contributing](/direct/contribute) guide.
```

Comment:
Thanks!

How about
- using explicit relative path (no leading forward slash)
- keeping the file extension (.md)

so that the link format is consistent with many other places?